doc/api/process.md

```diff
@@ -530,16 +530,14 @@ running the `./configure` script.
 An example of the possible output looks like:
 
 ```js
-{
-  target_defaults:
+{ target_defaults:
```

Comment:
I've changed these fragments as they are claimed to be the actual output — and this is the actual output. I prefer to update them than to leave inconsistent with other places with trailing commas.